#!/usr/bin/env python
PACKAGE = "get_pose"
NODE_NAME = "D2local_m2_dyn"
PARAMS_NAME = "MtwoParam"

from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lin_vel", double_t,  0, "Liner Velocity of robot in m/s - default value is 0.1.",
        0.1, 0.01, 1)
gen.add("psi_vel", double_t,  0, "Angular Velocity of robot in m/s - default value is 0.2.",
        0.2, 0.01, 0.4)
gen.add("gtol", double_t,  0, "Liner Velocity of robot in m/s - default value is 0.02.",
        0.02, 0.001, 0.1)
gen.add("psi_gtol", double_t,  0, "Liner Velocity of robot in m/s - default value is 0.1.",
        0.1, 0.01, 0.5)

#goal_tol=0.02, goal_tol_psi=0.1,

exit(gen.generate(PACKAGE, NODE_NAME, PARAMS_NAME))
